### ISSUE TITLE
Launch Plan (Marketing): Notebooks & PostHog 3000

### ISSUE THREAD

**Issue by joethreepwood** [PostHog/meta#139]
## What's launching, and when?
We're planning to launch a reduced-scope version of PostHog 3000, called 'Base 3000', which will launch alongside notebooks. This follows a period of internal testing.

The plan is to enable this feature as beta for 50% of teams, and run this as an experiment for a two week sprint. If nothing breaks? Full launch and announcement. All users who join the beta will get a new, in-app announcement. 

The dates may drift as the situation develops, but the plan is to enable the beta the week of 4 December, and initiate full release on the week of 18 Dec.

## Launch plans
The launch plan is divided into three stages. 

### Pre-launch, before the beta
- [x] Notebooks documentation - @ivanagas + @joethreepwood 
- [x] Social media teasers - (@andyvan-ph)

### Launch week to all users
- [x] In-app welcome message
- [x] Announcement blogpost detailing what's new in the new design https://github.com/PostHog/posthog.com/pull/7227 - @corywatilo 
- [x] Announcement blogpost introducing notebooks https://github.com/PostHog/posthog.com/pull/7252 - @joethreepwood 
- [x] [Walkthrough video](https://youtu.be/B73WnVRi0ko) (incl in blogpost) https://github.com/PostHog/posthog.com/issues/7137 - @ivanagas 
- [x] Announce to all users in the YC program and paying user Slack groups via Pylon - @joethreepwood 
- [x] Announce via all social channels - @andyvan-ph 
- [x] Add to Changelog - @joethreepwood 

### Follow-on activity
- [x] https://github.com/PostHog/meta/issues/165 - @joethreepwood 
- [x] Feature in Changelog & Startups email - @joethreepwood 
- [x] https://github.com/PostHog/meta/issues/164 - @joethreepwood 
- [x] https://github.com/PostHog/posthog.com/issues/7357 - @andyvan-ph 
- [x] Get a case study for notebooks - @joethreepwood 

## Updating the site
These are critical assets, which we should aim to update the _week of the full release_. Screenshots should be updated, or removed if not strictly needed. 

- [ ] Screenshots across the homepage and product pages - Website & Docs (@corywatilo / @smallbrownbike)
- [x] https://github.com/PostHog/posthog.com/pull/7354 - @joethreepwood 
- [x] https://github.com/PostHog/posthog.com/pull/7315) - @andyvan-ph 
- [x] https://github.com/PostHog/posthog.com/pull/7274 - @ivanagas 
- [ ] Screenshots across Docs - Engineering teams for each product.
- [x] https://github.com/PostHog/posthog.com/pull/7344 - @Lior539 
- [ ] Screenshots in Tutorials - @ivanagas 
- [ ] Simon's PostHog demo on /book-a-demo - @ivanagas & @simfish85 
- [x] Screenshots on G2.com - @joethreepwood 
- [x] Screenshots on LinkedIn- @joethreepwood 
- [x] Screenshots on Product Hunt - @joethreepwood

**Comment by andyvan-ph** [comment]
I know I suggest this every time, but I think this feature (more than most) would benefit from some kind of video demo / animation.

**Comment by joethreepwood** [comment]
> I know I suggest this every time, but I think this feature (more than most) would benefit from some kind of video demo / animation.

What sort of thing would you suggest, and how would we use it?

**Comment by benjackwhite** [comment]
Yeah I think so too. Question really ends up being "will the video stay up-to-date" but if we are purposeful about keeping it simple and easy to re-record then I'd be up for this.

**Comment by neilkakkar** [comment]
wellll, I will only say this kind of thing twice, but we do already have recordings for cypress tests, so technically, if you write a cypress test (or get chatGPT to write it xP), you could automatically generate a video out of it, and say update it every week/month with a click of a button. Not sure how useful this will be though, compared to a person making sure to highlight the important stuff.

**Comment by joethreepwood** [comment]
My question is still really about what we want to do. 

If we're thinking some sort of demo of some notebooks functionality to add in a blog post and embed in social announcements, etc, we can absolutely do that. And it won't matter too much if it ages because who's going back to those pages. 

The part I'm less keen on is doing some sort of polished explainer video which we can feature in more prominent positions on the site and re-use. That _will_ date and need to be kept up to date, and I'm not personally convinced of how valuable it would be compared to the effort required. I've worked on these projects before and rarely seen them end up worth it. 

So, yeah. Before we turn to AI to solve the problem, I want to hear a bit more about what the goal is and how we'd use a video so that we can actually create something that delivers on the objective, and we aren't doing video for the sake of it.

**Comment by andyvan-ph** [comment]
I think a 3 to 5 demo video from Ben, maybe with some light editing, would be fine. It's just a feature you really need to see in action. That plus a decent collection of short video clips would work nicely.

**Comment by joethreepwood** [comment]
I've updated the plan above, adjusting the date and adding the usual responsible people. Still open to feedback, and changes.

**Comment by corywatilo** [comment]
> where notebooks should feature in the docs and product hierarchy

Within Product OS for both

**Comment by joethreepwood** [comment]
After the standup last week I've updated the suggested launch plan to reflect the launch of Base 3000 and Notebooks together. Please take a look, share feedback, and add anything that I may be missing. 

If this looks good and no objections are raised in the marketing sprint, the next step will be putting owners next to items and starting to prep some of the messaging. 

We'll definitely want to have a video which walks through all of the 3000 changes, and demos notebooks. I'll create an issue to spec that out once we have a bit more clarity on what's Base 3000 will look like (I believe there'll be a demo on all-hands).

**Comment by simfish85** [comment]
> Simon's PostHog demo on /book-a-demo

Is this something I can get going on now / need to wait for further updates to be ready?  Should I also include notebooks in a demo?

**Comment by joethreepwood** [comment]
> > Simon's PostHog demo on /book-a-demo
> 
> Is this something I can get going on now / need to wait for further updates to be ready? Should I also include notebooks in a demo?

I'd wait for the internal testing period before aiming to create this, but I think yes to the second part (although it is not a paid feature)

**Comment by joethreepwood** [comment]
Quick update after catching up with Ben: 

- End of this week we hope to have it in a stable state
- We will do internal testing and start rolling out an experiment to all users to ensure we don't break anything / see our metrics drastically drop
- Means around first or second week of December we should be confident enough with the new design to do an announcement about it

The plan is now to do this gradual roll-out, rather than a big all-at-once announcement. We will still need much of the content above (video, tutorial refreshes, etc) so we shouldn't stop planning those - but I'm going to wait for more certainty on the impact before overhauling the plan.

**Comment by andyvan-ph** [comment]
@joethreepwood Just a reminder, I can sort the social media teasers, so no need to pick that up.

**Comment by andyvan-ph** [comment]
First teaser is live: https://twitter.com/posthog/status/1729192487573488048

Plan to do another tomorrow showing more notebook features. It'll be a reply to this tweet for better visibility.

**Comment by joethreepwood** [comment]
I’ve updated the top comment to be more specific, and to start putting people’s names next to things in terms of site updates. Please add to the issue if anything is wrong, or unrealistic. 

I’ve mostly added names based on what seems sensible (or is in progress already) and areas which individuals normally own, but this is just so we can stay coordinated - I’m nobody’s manager and I’m not trying to forcefully assign things to anyone.

**Comment by ivanagas** [comment]
@joethreepwood has forced my hand about updating PostHog 3000 tutorial images a bit with a couple PRs [like this one](https://github.com/PostHog/posthog.com/pull/7560) (fair enough 😅 ). I don't think it is better experience just to delete the old images without replacing them with new ones. Responding here instead of each of the PRs because others might want to weigh in on this. 

It is my responsibility to update them. I started doing it, but they were taking up a significant amount of my time over multiple days so dropped them. It takes a lot of time to recreate the setups I had at each stage of every tutorial and make sure the explanation around the images makes sense. I also hadn't heard a single piece of feedback saying that the tutorials used the old UI.

I also doubt the docs images will get updated by the teams without us forcing the issue, and they will run into the same trouble I had.

Three options:
1. I spend 1 week updating them as my main priority and get them done. 
2. We hire a freelancer to update them. Basically all of them can be recreated from context and image, although it will take them more time, and we will still have to review them all. 
3. We leave them as is as update them as we update the tutorials or people complain about them

I'd prefer not to do 1 but will do it if needed.

**Comment by joethreepwood** [comment]
Wasn't meaning to force your hand - just fixing them as I noticed them while I did some other docs work for https://github.com/PostHog/meta/issues/171

I definitely don't think we should wait for 3. People won't actively complain about them as often as they'll go to check the docs, get confused, get irritated, and move on. 

I also agree that tech teams won't fix this, even though I've tried to force it. 

There is a fourth option: Get a digital asset management (DAM) tool in place to make it easier to update and maintain these images. I don't have a strong opinion here (@corywatilo and @smallbrownbike may though), other than that in my experience these are prohibitively expensive. 

I'll add this to the marketing sprint issue so we can discuss, but I'm definitely in favour of getting a freelancer in here.
